tests/cases/standalone/common/basic.sql

```diff
@@ -47,6 +47,15 @@ INSERT INTO t1 VALUES ('host1',0, 0), ('host2', 1, 1,);
 
 SELECT * from t1;
 
+-- issue #4649 should fail (do not support delete from logical table for now)
+delete from t1;
+
+-- issue #4649 should succeed
+delete from phy;
```

Comment:
Do we need to select the table again?
```suggestion
delete from phy;

SELECT * from t1;
```

Reply:
Yes t1 affects by the phy

Reply:
We can select t1 again to verify it is empty.

Reply:
FYI: there's another select t1 below in the sql file
and I can just drop the insert